splinepy/multipatch.py

```diff
@@ -17,11 +17,7 @@ class Multipatch(SplinepyBase, PyMultiPatch):
 
     __slots__ = ()
 
-    def __init__(
-        self,
-        splines=None,
-        interfaces=None,
-    ):
+    def __init__(self, splines=None, interfaces=None, *, spline=None):
```

Comment:
What is the-* used for?

Reply:
makes sure `spline` is only set as keyword. say, `Multipatch(splines, interfaces, garbage)` will raise TypeError regarding number of positional argument